tests/parametric/test_otel_span_methods.py

```diff
@@ -582,3 +790,25 @@ def run_otel_span_reserved_attributes_overrides_analytics_event(
     else:
         assert "_dd1.sr.eausr" not in span["metrics"]
     assert "analytics.event" not in span["meta"]
+
+
+def retrieve_span_links(span):
+    if span.get("span_links") is not None:
+        return span["span_links"]
+    elif span["meta"].get("_dd.span_links") is not None:
+        # Convert span_links tags into protobuf v0.4 format
+        json_links = json.loads(span["meta"].get("_dd.span_links"))
+        links = []
+        for json_link in json_links:
+            link = {}
+            link["trace_id"] = int(json_link["trace_id"][-16:], base=16)
+            link["trace_id_high"] = int(json_link["trace_id"][:16], base=16) if len(json_link["trace_id"]) > 16 else 0
+            link["span_id"] = int(json_link["span_id"], base=16)
+            link["attributes"] = json_link.get("attributes") or {}
+            link["tracestate"] = json_link.get("tracestate") or ""
+            # If set, the high bit (bit 31) should be set according the RFC
+            link["flags"] = 0 if json_link.get("flags") is None else (json_link.get("flags") | -2147483648)
```

Comment:
Naive question: Traceflags are encoded as unsigned 32bit integers. Why are using `-1 << 31` to set the high bit. Shouldn't we use `1 << 31`?

Reply:
Not so naive question... That is something I borrowed from @bwoebi because I don't use it when encoded through tags. 
You can find here: https://github.com/DataDog/system-tests/pull/1811/files#diff-e63af22f5cd81bf600c30dfb393aef024e358dd336ede3786d147f929cab2b54R145

@bwoebi Can you provide guidance here?

Reply:
`1 | ( -1 <<31)` has the binary representation of `-0b1111111111111111111111111111111` (instead of 
`0b10000000000000000000000000000001`)

I think using `-1 << 31` to set the highest order bit is incorrect. We should use `1 << 31`. 

@bwoebi Is the `flags` field set in the php parametric app? We could be consistently hitting the `0 if json_link.get("flags")`  case. This would return the expected value in these tests:   https://github.com/DataDog/system-tests/pull/1811/files#diff-e63af22f5cd81bf600c30dfb393aef024e358dd336ede3786d147f929cab2b54R145

Reply:
I created a dedicated constant so we can fix it later for any test once you find the right value to use:
```python
from utils.parametric.spec.tracecontext import TRACECONTEXT_FLAGS_SET
```